nextcloud_update.sh

```diff
@@ -552,6 +565,17 @@ fi
 # Recover apps that exists in the backed up apps folder
 run_script STATIC recover_apps
 
+# Restore app groups
+print_text_in_color "$ICyan" "Restoring all groups for enabled apps. This can take a while..."
+for app in "${!APPSTORAGE[@]}"
+do 
+    if ! is_app_enabled "$app"
+    then
+        occ_command_no_check app:enable "$app"
+    fi
```

Comment:
as a note to myself: if storing all app configuration, this is needed to be removed.